upup/pkg/fi/cloudup/template_functions.go

```diff
@@ -90,6 +90,9 @@ func (tf *TemplateFunctions) WellKnownServiceIP(id int) (net.IP, error) {
 func (tf *TemplateFunctions) AddTo(dest template.FuncMap) {
 	dest["SharedVPC"] = tf.SharedVPC
 
+	// Remember that we may be on a different arch from the target.  Hard-code for now.
+	dest["Arch"] = func() string { return "amd64" }
```

Comment:
I am wondering if we should start defining the arch that kops is installing to, part of the cluster spec.